/var/www/rbfaq/forum/settings/base.py TIME: 2010-08-28 03:52:49,404 MSG: base.py:value:65 Error retrieving setting from database (FORM_EMPTY_QUESTION_BODY): (1146, "Table 'osqa.forum_keyvalue' doesn't exist")
/var/www/rbfaq/forum/settings/base.py TIME: 2010-08-28 03:52:49,406 MSG: base.py:value:65 Error retrieving setting from database (FORM_MIN_NUMBER_OF_TAGS): (1146, "Table 'osqa.forum_keyvalue' doesn't exist")
/var/www/rbfaq/forum/settings/base.py TIME: 2010-08-28 03:52:49,407 MSG: base.py:value:65 Error retrieving setting from database (FORM_MAX_NUMBER_OF_TAGS): (1146, "Table 'osqa.forum_keyvalue' doesn't exist")
/var/www/rbfaq/forum/settings/base.py TIME: 2010-08-28 03:52:49,408 MSG: base.py:value:65 Error retrieving setting from database (FORM_EMPTY_QUESTION_BODY): (1146, "Table 'osqa.forum_keyvalue' doesn't exist")
/var/www/rbfaq/forum/settings/base.py TIME: 2010-08-28 03:52:49,423 MSG: base.py:value:65 Error retrieving setting from database (MIN_USERNAME_LENGTH): (1146, "Table 'osqa.forum_keyvalue' doesn't exist")
/var/www/rbfaq/forum/modules/__init__.py TIME: 2010-08-28 03:52:49,655 MSG: __init__.py:get_modules_script:35 Error importing badges from module forum_modules.default_badges: 
 Traceback (most recent call last):
  File "/var/www/rbfaq/forum/modules/__init__.py", line 28, in get_modules_script
    all.append(__import__('%s.%s' % (m.__name__, script_name), globals(), locals(), [m.__name__]))
  File "/var/www/rbfaq/forum_modules/default_badges/badges.py", line 3, in <module>
    from forum.badges.base import AbstractBadge
  File "/var/www/rbfaq/forum/badges/base.py", line 12, in <module>
    installed = dict([(b.cls, b) for b in Badge.objects.all()])
  File "/usr/lib/pymodules/python2.6/django/db/models/query.py", line 106, in _result_iter
    self._fill_cache()
  File "/usr/lib/pymodules/python2.6/django/db/models/query.py", line 692, in _fill_cache
    self._result_cache.append(self._iter.next())
  File "/usr/lib/pymodules/python2.6/django/db/models/query.py", line 238, in iterator
    for row in self.query.results_iter():
  File "/usr/lib/pymodules/python2.6/django/db/models/sql/query.py", line 287, in results_iter
    for rows in self.execute_sql(MULTI):
  File "/usr/lib/pymodules/python2.6/django/db/models/sql/query.py", line 2369, in execute_sql
    cursor.execute(sql, params)
  File "/usr/lib/pymodules/python2.6/django/db/backends/mysql/base.py", line 84, in execute
    return self.cursor.execute(query, args)
  File "/usr/lib/pymodules/python2.6/MySQLdb/cursors.py", line 166, in execute
    self.errorhandler(self, exc, value)
  File "/usr/lib/pymodules/python2.6/MySQLdb/connections.py", line 35, in defaulterrorhandler
    raise errorclass, errorvalue
ProgrammingError: (1146, "Table 'osqa.forum_badge' doesn't exist")

/var/www/rbfaq/forum/templatetags/ui_registry.py TIME: 2010-08-28 04:01:43,924 MSG: ui_registry.py:render:26 Exception 'request' rendering ui objects <forum.modules.ui_objects.Link object at 0xba03a38c>: 
Traceback (most recent call last):
  File "/var/www/rbfaq/forum/templatetags/ui_registry.py", line 22, in render
    result += ui_object.render(context)
  File "/var/www/rbfaq/forum/modules/ui_objects.py", line 107, in render
    html.hyperlink(self.url(context), self.text(context), **self.attrs(context)),
  File "/var/www/rbfaq/forum/modules/ui_objects.py", line 67, in __call__
    return self.argument(user, context)
  File "/var/www/rbfaq/forum/registry.py", line 41, in <lambda>
    url=lambda u, c: settings.CONTACT_URL and settings.CONTACT_URL or "%s?next=%s" % (reverse('feedback'), c['request'].path),
  File "/usr/lib/pymodules/python2.6/django/template/context.py", line 44, in __getitem__
    raise KeyError(key)
KeyError: 'request'

/var/www/rbfaq/forum/templatetags/ui_registry.py TIME: 2010-08-28 09:10:33,561 MSG: ui_registry.py:render:26 Exception 'request' rendering ui objects <forum.modules.ui_objects.Link object at 0xba0f4dec>: 
Traceback (most recent call last):
  File "/var/www/rbfaq/forum/templatetags/ui_registry.py", line 22, in render
    result += ui_object.render(context)
  File "/var/www/rbfaq/forum/modules/ui_objects.py", line 107, in render
    html.hyperlink(self.url(context), self.text(context), **self.attrs(context)),
  File "/var/www/rbfaq/forum/modules/ui_objects.py", line 67, in __call__
    return self.argument(user, context)
  File "/var/www/rbfaq/forum/registry.py", line 41, in <lambda>
    url=lambda u, c: settings.CONTACT_URL and settings.CONTACT_URL or "%s?next=%s" % (reverse('feedback'), c['request'].path),
  File "/usr/lib/pymodules/python2.6/django/template/context.py", line 44, in __getitem__
    raise KeyError(key)
KeyError: 'request'

/var/www/rbfaq/forum/templatetags/ui_registry.py TIME: 2010-08-28 09:10:33,620 MSG: ui_registry.py:render:26 Exception 'request' rendering ui objects <forum.modules.ui_objects.Link object at 0xba0f4dec>: 
Traceback (most recent call last):
  File "/var/www/rbfaq/forum/templatetags/ui_registry.py", line 22, in render
    result += ui_object.render(context)
  File "/var/www/rbfaq/forum/modules/ui_objects.py", line 107, in render
    html.hyperlink(self.url(context), self.text(context), **self.attrs(context)),
  File "/var/www/rbfaq/forum/modules/ui_objects.py", line 67, in __call__
    return self.argument(user, context)
  File "/var/www/rbfaq/forum/registry.py", line 41, in <lambda>
    url=lambda u, c: settings.CONTACT_URL and settings.CONTACT_URL or "%s?next=%s" % (reverse('feedback'), c['request'].path),
  File "/usr/lib/pymodules/python2.6/django/template/context.py", line 44, in __getitem__
    raise KeyError(key)
KeyError: 'request'

/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,069 MSG: base.py:value:65 Error retrieving setting from database (APP_SHORT_NAME): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,262 MSG: base.py:value:65 Error retrieving setting from database (GOOGLE_SITEMAP_CODE): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,298 MSG: base.py:value:65 Error retrieving setting from database (USE_CUSTOM_CSS): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,302 MSG: base.py:value:65 Error retrieving setting from database (USE_CUSTOM_HEAD): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,303 MSG: base.py:value:65 Error retrieving setting from database (APP_SHORT_NAME): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,305 MSG: base.py:value:65 Error retrieving setting from database (USE_CUSTOM_HEADER): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,363 MSG: base.py:value:65 Error retrieving setting from database (APP_LOGO): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,373 MSG: base.py:value:65 Error retrieving setting from database (APP_TITLE): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,413 MSG: base.py:value:65 Error retrieving setting from database (USE_ANNOUNCEMENT_BAR): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,523 MSG: base.py:value:65 Error retrieving setting from database (FAQ_PAGE_TEXT): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,848 MSG: base.py:value:65 Error retrieving setting from database (REP_GAIN_BY_UPVOTED): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,870 MSG: base.py:value:65 Error retrieving setting from database (REP_LOST_BY_DOWNVOTED): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,920 MSG: base.py:value:65 Error retrieving setting from database (REP_TO_COMMENT): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,921 MSG: base.py:value:65 Error retrieving setting from database (REP_TO_DELETE_COMMENTS): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,939 MSG: base.py:value:65 Error retrieving setting from database (REP_TO_CLOSE_OWN): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,957 MSG: base.py:value:65 Error retrieving setting from database (REP_TO_REOPEN_OWN): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,975 MSG: base.py:value:65 Error retrieving setting from database (REP_TO_RETAG): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,977 MSG: base.py:value:65 Error retrieving setting from database (REP_TO_EDIT_OTHERS): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,978 MSG: base.py:value:65 Error retrieving setting from database (REP_TO_CLOSE_OTHERS): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:47,979 MSG: base.py:value:65 Error retrieving setting from database (REP_TO_DELETE_COMMENTS): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:48,073 MSG: base.py:value:65 Error retrieving setting from database (USE_CUSTOM_FOOTER): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-02-27 22:08:48,108 MSG: base.py:value:65 Error retrieving setting from database (GOOGLE_ANALYTICS_KEY): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-04 15:20:37,457 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-04 21:03:07,085 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-04 23:40:22,776 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-05 05:41:51,128 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-05 18:46:13,980 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-07 16:07:18,603 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-10 01:57:25,994 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-11 05:19:51,569 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-12 10:53:43,060 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-18 19:42:25,079 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-19 01:11:49,221 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-19 11:43:54,644 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-19 12:31:44,111 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-19 16:44:03,514 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-21 06:27:31,547 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-21 06:27:35,404 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-21 07:24:24,942 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-21 07:24:28,232 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-24 13:28:49,963 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-26 16:21:27,648 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-03-28 19:59:50,890 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-05 16:37:33,192 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-05 23:58:25,573 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-12 02:42:37,675 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-13 09:39:54,361 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-13 18:20:37,743 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-15 23:00:12,066 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-16 04:08:22,259 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-16 08:55:31,131 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-17 19:44:29,889 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-22 06:53:15,165 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-22 23:20:46,966 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-24 05:09:55,954 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-25 05:21:19,672 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-27 23:59:35,014 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-28 17:24:49,749 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-29 05:08:53,839 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-30 04:24:11,147 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-04-30 09:21:38,199 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-02 04:51:52,706 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-15 12:06:03,367 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-15 12:14:06,197 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-15 12:22:09,936 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-18 04:27:16,169 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-18 06:44:49,845 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-18 16:00:21,599 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-18 16:51:52,675 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-18 18:06:24,971 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-20 04:31:44,567 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-21 01:22:50,648 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-21 18:59:02,417 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-21 19:12:41,773 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-23 10:20:32,160 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-23 20:59:18,425 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-24 22:42:54,237 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-25 11:39:42,211 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/settings/base.py TIME: 2011-05-27 22:55:10,649 MSG: base.py:value:65 Error retrieving setting from database (FORM_EMPTY_QUESTION_BODY): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-05-27 22:55:10,661 MSG: base.py:value:65 Error retrieving setting from database (FORM_MIN_NUMBER_OF_TAGS): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-05-27 22:55:10,712 MSG: base.py:value:65 Error retrieving setting from database (FORM_MAX_NUMBER_OF_TAGS): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-05-27 22:55:10,713 MSG: base.py:value:65 Error retrieving setting from database (FORM_EMPTY_QUESTION_BODY): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/settings/base.py TIME: 2011-05-27 22:55:10,748 MSG: base.py:value:65 Error retrieving setting from database (MIN_USERNAME_LENGTH): (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")
/var/www/rbfaq/forum/modules/__init__.py TIME: 2011-05-27 22:55:10,967 MSG: __init__.py:get_modules_script:35 Error importing badges from module forum_modules.default_badges: 
 Traceback (most recent call last):
  File "/var/www/rbfaq/forum/modules/__init__.py", line 28, in get_modules_script
    all.append(__import__('%s.%s' % (m.__name__, script_name), globals(), locals(), [m.__name__]))
  File "/var/www/rbfaq/forum_modules/default_badges/badges.py", line 3, in <module>
    from forum.badges.base import AbstractBadge
  File "/var/www/rbfaq/forum/badges/base.py", line 12, in <module>
    installed = dict([(b.cls, b) for b in Badge.objects.all()])
  File "/usr/lib/pymodules/python2.6/django/db/models/query.py", line 106, in _result_iter
    self._fill_cache()
  File "/usr/lib/pymodules/python2.6/django/db/models/query.py", line 692, in _fill_cache
    self._result_cache.append(self._iter.next())
  File "/usr/lib/pymodules/python2.6/django/db/models/query.py", line 238, in iterator
    for row in self.query.results_iter():
  File "/usr/lib/pymodules/python2.6/django/db/models/sql/query.py", line 287, in results_iter
    for rows in self.execute_sql(MULTI):
  File "/usr/lib/pymodules/python2.6/django/db/models/sql/query.py", line 2368, in execute_sql
    cursor = self.connection.cursor()
  File "/usr/lib/pymodules/python2.6/django/db/backends/__init__.py", line 81, in cursor
    cursor = self._cursor()
  File "/usr/lib/pymodules/python2.6/django/db/backends/mysql/base.py", line 281, in _cursor
    self.connection = Database.connect(**kwargs)
  File "/usr/lib/pymodules/python2.6/MySQLdb/__init__.py", line 81, in Connect
    return Connection(*args, **kwargs)
  File "/usr/lib/pymodules/python2.6/MySQLdb/connections.py", line 170, in __init__
    super(Connection, self).__init__(*args, **kwargs2)
OperationalError: (2002, "Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)")

/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-29 22:48:33,163 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-31 01:33:35,772 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-05-31 01:59:52,686 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-02 01:13:41,284 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-10 10:17:59,895 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-11 12:59:21,469 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-12 00:16:17,155 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-12 04:33:06,772 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-12 08:48:53,963 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-13 17:04:33,351 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-14 05:04:33,914 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-14 05:22:19,671 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-14 05:37:46,073 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-14 22:46:17,114 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-16 03:48:05,510 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-23 19:04:10,336 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-26 15:58:00,181 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-27 05:23:09,167 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-27 23:22:05,606 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-28 12:18:29,956 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-29 15:26:43,279 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-29 22:39:27,190 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-30 16:37:16,812 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-06-30 22:51:16,700 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/未回答/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-01 05:28:21,651 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-02 11:32:06,698 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-02 12:15:42,356 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-02 12:29:52,562 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-02 12:48:34,479 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-02 16:43:03,896 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-02 20:27:29,942 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-02 22:31:19,804 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-06 14:50:55,273 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-06 18:11:39,409 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-06 18:37:59,697 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-07 01:31:52,510 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-07 01:52:08,267 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-07 03:18:32,003 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-08 08:19:05,748 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-10 00:25:36,107 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-10 07:51:30,435 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-10 23:20:39,489 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-12 23:43:55,241 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-13 06:59:07,693 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-13 11:42:11,369 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-13 20:44:56,840 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-14 11:53:26,036 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-14 11:59:10,024 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-14 12:25:19,854 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-14 19:54:46,776 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-07-17 01:15:48,779 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-08-02 18:01:11,600 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-08-03 06:01:16,999 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/unanswered/, refered by UNKNOWN
/var/www/rbfaq/forum/utils/pagination.py TIME: 2011-08-04 21:33:25,663 MSG: pagination.py:_paginated:184 Found invalid page number "2", loading /questions/, refered by UNKNOWN
